.eslintrc.js

```diff
@@ -850,6 +850,10 @@ module.exports = {
                 name: 'semver',
                 message: 'Please use "semver/*/{function}" instead',
               },
+              {
+                name: '@kbn/rule-data-utils',
+                message: `Import directly from @kbn/rule-data-utils/* submodules in public/common code`,
```

Comment:
Is this message specific to `rule-data-utils`?

Should this read generically like:
```ts
Import directly from @kbn/<module_name>/* in public/common code
```

or
```ts
Import directly from @kbn/*/* in public/common code
```

Reply:
Yes, this is specific to the `@kbn/rule-data-utils` package as import targets have to be setup for each package individually